source/extensions/filters/http/header_to_metadata/header_to_metadata_filter.cc

```diff
@@ -113,6 +114,18 @@ bool HeaderToMetadataFilter::addMetadata(StructMap& map, const std::string& meta
     }
     break;
   }
+  case envoy::config::filter::http::header_to_metadata::v2::Config_ValueType_BASE64URL_SERIALIZED: {
+    const auto decoded = Base64Url::decode(std::string(value));
```

Comment:
There is a string_view version of this I think.

Reply:
Hmm, I didn't find it in https://github.com/envoyproxy/envoy/blob/350a35f2ac685c4958934466a90b40f876968f99/source/common/common/base64.h#L63, mind to give a link?

Reply:
Base64 decodeWithoutPadding. Do you need Base64URL or Base64 encoding? I don't think it matters in a header AFAICT.

Reply:
Thanks! I changed to use this but I think we need to construct the string anyway for other value types.